### ISSUE TITLE
Licence recommandée par défaut

### ISSUE THREAD

**Issue by pajachiet** [entrepreneur-interet-general/eig-link#76]
Sur la page [opensource.md](https://github.com/entrepreneur-interet-general/eig-link/blob/master/opensource.md), on lit :

> Si vous publiez un code source, il est recommandé de publier votre dépôt sous la licence MIT.

Pourquoi est-ce le choix par défaut et pas une licence copyleft ?

**Comment by bzg** [comment]
Oui, cette recommandation est problématique, nous allons proposer une reformulation.

**Comment by AntoineAugusti** [comment]
Hésite pas à proposer une PR @bzg et discutons-en.

**Comment by bzg** [comment]
Voici la PR: https://github.com/entrepreneur-interet-general/eig-link/pull/77

L'idée générale est que la DINSIC ne recommande pas de licence par défaut : chaque cas est un cas particulier sur lequel Etalab propose un accompagnement, j'ai indiqué les personnes à solliciter.

La PR contient aussi une précision sur la phrase introduisant les licences homologuées: elles portent sur le code _nouvellement produit_, distinct du code "publié", lequel peut contenir des morceaux de code hérité d'autres projets libres, placés sous des licences autres que celles homologuées.

**Comment by AntoineAugusti** [comment]
Vu dans PR #77
